### PR TITLE
Update documentation from Content to Collection Type

### LINKED ISSUE #5555: References to old name "content-type" in documentation and UI
"content type" is now "collection type" but the UI and documentation still has many references to "content types".

**Steps to reproduce**

1. Go to [Quick Start Guide](https://strapi.io/documentation/3.0.0-beta.x/getting-started/quick-start.html#_2-create-an-administrator-user) and scroll down to "3. Create a Restaurant Content Type"
2. The first step says 'Click the "+ Create new content-type" link'. Should say "+ 'Create new collection type" link'.

**Steps to reproduce**
1. Log into the Admin site
2. On the left sidebar, under PLUGINS, the "Content-Types Builder" should say "Collection-Types Builder"

### PR DESCRIPTION
**Pull request by adriatic**
Update tutorial files to use the Collection type instead of Content type

#### Description of what you did:
Updated tutorial files **`quick-start.md`** and **`quick-start-tutorial.md`** to use the Collection type instead of Content type

close #5555

### CODE REVIEW

**Comment by christianguevara** [comment]
Related with https://github.com/strapi/strapi/pull/5635

**Comment by adriatic** [comment]
@christianguevara - [your comment above](https://github.com/strapi/strapi/pull/5725#issuecomment-609883410) needs a brief explanation: I started my task to just verify the @chinhdo [Pull Request](https://github.com/strapi/strapi/pull/5635) - and found too many places that need still do be changed, to just update his PR; so @chinhdo suggested to create a new one that starts with the current sources on the master

**Comment by christianguevara** [comment]
Well, I didn't knew you checked the PR https://github.com/strapi/strapi/pull/5635 because you didn't said so. Then there's no problem I guess. I was trying to avoid duplications of work in the same thing :)

**Comment by adriatic** [comment]
@christianguevara - when I found that (**already merged** https://github.com/strapi/strapi/pull/5635) has still many instances of **`Content Type`** where it should be **`Collection Type`**, I briefly spoke with @chinhdo and @derrickmehaffy - that yielded the proposal to go ahead, fix them all and create a new PR.

Hope it makes sense.

**Comment by christianguevara** [comment]
For sure, thank you :)

**Comment by chinhdo** [comment]
Hi, There may be instances where the use of "content type" is appropriate? If it's referring to any content type (collection type or single type).

For example in this statement: "Both Collection Type and Single Type are types of **Content  Type**, the use of "Content Type" is appropriate there.

**Comment by adriatic** [comment]
@chinhdo ref [your comment])https://github.com/strapi/strapi/pull/5725#issuecomment-609976765) - I believe that I handled all such instances, yes. As you can see [my PR](https://github.com/strapi/strapi/pull/5725) needs review, so of you find a case like you described, please let us now here in this thread

**Comment by chinhdo** [comment]
@adriatic I did see this change in this PR:

> Both **Collection Type** and **Single Type** are types of **Collection Type**. **Collection Type** is used for content that repeats such as restaurants, or blog posts (hence "collection"). **Single Type** is used to manage content that will have only one instance - for example, your menu, a static page, or a footer.

I'll try to take a closer look later today. Thanks.

**Comment by chinhdo** [comment]
Yes ... because that's the purpose of that sentence, to explain that both Collection Type and Single Type are types of Content Type. As far as I know there isn't any other Content Type other than Collection Type and Single Type at this time.

Here's a [blog post](https://strapi.io/blog/beta-19-single-types-uid-field) that talks about the introduction of *Single Type*. Thanks.

**Comment by adriatic** [comment]
Thanks for the blog reference. I did see that article few days ago and it did not prevent me from writing **`Both Collection Type and Single Type are types of Collection Type`**. While logically (mathematically) this version is not wrong (_each set contains itself as a subset_), I believe not that **`Both Collection Type and Single Type are types of Content Type`** is better.

Thank you for pursuing this :-)

File: docs/3.0.0-beta.x/getting-started/quick-start-tutorial.md [hunk 1]
```diff
@@ -190,39 +190,39 @@ You are now ready to create your first **Collection Type**.
 
 A **Content Type** can be considered a sort of _blueprint_ for the data created. In other words, a **Content Type** is the schema of the data structure.
 
-Both **Collection Type** and **Single Type** are types of **Content Type**. **Collection Type** is used for content that repeats such as restaurants, or blog posts (hence "collection"). **Single Type** is used to manage content that will have only one instance - for example, your menu, a static page, or a footer.
+Both **Collection Type** and **Single Type** are types of **Collection Type**. **Collection Type** is used for content that repeats such as restaurants, or blog posts (hence "collection"). **Single Type** is used to manage content that will have only one instance - for example, your menu, a static page, or a footer.
```
Comment (lauriejim) [inline docs/3.0.0-beta.x/getting-started/quick-start-tutorial.md:193]: Here it should be `Content Type`

Both **Collection Type** and **Single Type** are types of **Content Type**

File: docs/3.0.0-beta.x/getting-started/quick-start-tutorial.md [hunk 2]
```diff
@@ -577,7 +578,7 @@ You have now entered your first **Restaurant** Content Type. You have also assig
 
 ## 7. Set Roles and Permissions
 
-By default, Strapi publishes all **Content Types** with restricted permissions. Which means you have to explicitly give permissions to each **Content Type** you create. You are going to give **Public** API (or URL) access to both the **Restaurant** Content Type and **Category** Content Type.
+By default, Strapi publishes all **Collection Types** with restricted permissions. Which means you have to explicitly give permissions to each **Collection Type** you create. You are going to give **Public** API (or URL) access to both the **Restaurant** Collection Type and **Category** Collection Type.
```
Comment (lauriejim) [inline docs/3.0.0-beta.x/getting-started/quick-start-tutorial.md:581]: Here you have to use `Content Type` for the 2 first.
Both Collection and Single Types have restrictions.

By default, Strapi publishes all **Content Types** with restricted permissions. This means you have to explicitly give permissions to each **Content Type** you create. You are going to give **Public** API (or URL) access to both the **Restaurant** Collection Type and **Category** Collection Type.

File: docs/3.0.0-beta.x/getting-started/quick-start-tutorial.md [hunk 3]
```diff
@@ -622,7 +623,7 @@ Forbidden Access Looks like this:
 
 ::: tip NOTE
 
-If you would like to see the route of any specific **Content Type**, you need to navigate to the **Content Type** under the **Roles and Permissions** plugin and click the ⚙️ next to the value. On the right, you see the route:
+If you would like to see the route of any specific **Collection Type**, you need to navigate to the **Collection Type** under the **Roles and Permissions** plugin and click the ⚙️ next to the value. On the right, you see the route:
```
Comment (lauriejim) [inline docs/3.0.0-beta.x/getting-started/quick-start-tutorial.md:626]: You should use `Content Type` in this sentence, this is a tip about a global concept that concern Collection and Single Types.

Comment (adriatic) [reply docs/3.0.0-beta.x/getting-started/quick-start-tutorial.md:626]: All requested changes are very obvious to the point that I do not understand how I missed them.

**Comment by adriatic** [comment]
@lauriejim I believe that I satisfied your **[last request above](https://github.com/strapi/strapi/pull/5725#discussion_r406659346)** and pushed the changes to the few days old PR (**[branch adriatic](https://github.com/adriatic/strapi/tree/adriatic)** on my clone of the strapi repo).

My experience in patching a PR is reasonably limited, so, the above warning:

image

**_might_** cause harm to Strapi repo. (I think it is asking to merge the current Strapi master branch to adriatic branch)

**Comment by derrickmehaffy** [comment]
> @lauriejim I believe that I satisfied your **[last request above](https://github.com/strapi/strapi/pull/5725#discussion_r406659346)** and pushed the changes to the few days old PR (**[branch adriatic](https://github.com/adriatic/strapi/tree/adriatic)** on my clone of the strapi repo).
> 
> My experience in patching a PR is reasonably limited, so, the warning below
> 
> image
> 
> might cause harm to Strapi repo. (I think it is asking to merge the current Strapi master branch to `adriatic` branch)
> 
> Your whole GitHub setup is quite impressive, not to mention the Strapi code and functionality. So, I feel privileged to be able to help.

@adriatic it will probably be easier for you to just rebase off the master branch (typically what I do when working on large PRs)

File: docs/3.0.0-beta.x/getting-started/quick-start-tutorial.md (hunk 3 above)
Comment (lauriejim) [reply docs/3.0.0-beta.x/getting-started/quick-start-tutorial.md]: No problem, thank for the update. I really appreciate.

**Pull request merged**